certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7710 @@
+        "description": "Deletes the OBS unit specified by the given ID. If the OBS unit with the given ID does not exist, the request will fail.",
+        "operationId": "DeleteObsUnit",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "obsTypeId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+            "x-ms-summary": "ObsTypeId",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "obsUnitId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "6f4e2c60-1372-4fe2-b009-8c4ddd510213",
+            "x-ms-summary": "ObsUnitId",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content",
+            "schema": {
+              "$ref": "#/definitions/Unit"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        },
+        "security": [
+          {
+            "API-Token": []
+          },
+          {
+            "OAuth": []
+          }
+        ]
+      },
+      "patch": {
+        "tags": [
+          "OBS"
+        ],
+        "summary": "Update an OBS Unit",
+        "description": "Updates a single OBS unit's name or moves it to a different parent.",
+        "operationId": "UpdateObsUnit",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "obsTypeId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+            "x-ms-summary": "ObsTypeId",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "obsUnitId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "6f4e2c60-1372-4fe2-b009-8c4ddd510213",
+            "x-ms-summary": "ObsUnitId",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "body",
+            "name": "obsUnitUpdateRequest",
+            "description": "ObsUnit represented by a JSON document.",
```

Comment:
Since these are technical terms of our product, we assume that clients from all locales should be familiar with them, and they don't need to be translated.